synapse/cortex.py

```diff
@@ -1297,6 +1321,8 @@ async def initServiceStorage(self):
             (1, self._storUpdateMacros),
         ), nexs=False)
 
+        self._initVaults()
```

Comment:
Should this be done **before** we run the `cortex:storage` update functions?

Reply:
¯\_( ͡° ͜ʖ ͡°)_/¯

You mean the `_bumpCellVers()` call right above this? I have no idea. If you're asking, the answer is probably yes.